src/lm/arpa-file-parser.cc

```diff
@@ -38,6 +38,12 @@ ArpaFileParser::ArpaFileParser(ArpaParseOptions options,
 ArpaFileParser::~ArpaFileParser() {
 }
 
+std::string rtrim(const std::string &str) {
```

Comment:
... actually, why not make it take a pointer to string and return void, and just use resize to remove the trailing whitespace?